build/ci.go

```diff
@@ -173,6 +173,21 @@ func doInstall(cmdline []string) {
 	if flag.NArg() > 0 {
 		packages = flag.Args()
 	}
+
+	if runtime.GOOS == "windows" {
+		// Resolve ./... manually and remove vendor/bazil/fuse (fuse is not in windows)
+		out, err := goTool("list", "./...").CombinedOutput()
+		if err != nil {
+			log.Fatalf("package listing failed: %v\n%s", err, string(out))
+		}
+		packages = []string{}
+		for _, line := range strings.Split(string(out), "\n") {
+			if !strings.Contains(line, "bazil") {
+				packages = append(packages, strings.TrimSpace(line))
+			}
+		}
+	}
```

Comment:
If I understand correctly, this change fixes `make all` and related commands by excluding "bazil.org/fuse" from the expansion of "./..." for the purpose of building and testing.

My suggestion is that all vendored packages should be excluded. Building vendored packages explicitly is almost never needed; testing them doesn't do anything because we strip their _test.go files when adding them. They will still be built as dependencies of non-vendored packages.